`make` called to create reference manual fails if `build/generated/doc` is missing

try this:
```
 > rm -rf build/generated/doc
 > make build/doc/reference_manual/fuzion_reference_manual.pdf 
mkdir -p build/generated/doc
cd . && git log modules/base/src/encodings/unicode/data.fz  | grep --extended-regexp "^Date:" | head | sed "s-Date:   -:UNICODE_VERSION: -g" | head -n1 > /unicode_version.adoc
/bin/sh: 1: cannot create /unicode_version.adoc: Permission denied
make: *** [Makefile:1230: build/generated/doc/unicode_version.adoc] Fehler 2
```
It looks as if the call to `$(realpath $(@D))` returns the empty string if `$(@D)` does not exist. 
A second call to `make` works: 
```
 > make build/doc/reference_manual/fuzion_reference_manual.pdf 
mkdir -p build/generated/doc
cd . && git log modules/base/src/encodings/unicode/data.fz  | grep --extended-regexp "^Date:" | head | sed "s-Date:   -:UNICODE_VERSION: -g" | head -n1 > /home/fridi/fuzion/fuzion/build/generated/doc/unicode_version.adoc
mkdir -p build/generated/doc
[...]
```

--- a/docs.mk
+++ b/docs.mk
@@ -86,7 +86,7 @@ $(DOC_JAVA): $(JAVA_FILE_UTIL_VERSION) $(JAVA_FILE_FUIR_ANALYSIS_ABSTRACT_INTERP
 
 $(BUILD_DIR)/generated/doc/unicode_version.adoc:
 	mkdir -p $(@D)
-	cd $(FZ_SRC) && git log modules/base/src/encodings/unicode/data.fz  | grep --extended-regexp "^Date:" | head | sed "s-Date:   -:UNICODE_VERSION: -g" | head -n1 > $(realpath $(@D))/unicode_version.adoc
+	cd $(FZ_SRC) && git log modules/base/src/encodings/unicode/data.fz  | grep --extended-regexp "^Date:" | head | sed "s-Date:   -:UNICODE_VERSION: -g" | head -n1 > $$(realpath $(@D))/unicode_version.adoc
 
 $(BUILD_DIR)/generated/doc/codepoints_white_space.adoc: $(CLASS_FILES_PARSER)
 	mkdir -p $(@D)

--- a/docs.mk
+++ b/docs.mk
@@ -84,9 +84,11 @@ REF_MANUAL_ATTRIBUTES = \
 $(DOC_JAVA): $(JAVA_FILE_UTIL_VERSION) $(JAVA_FILE_FUIR_ANALYSIS_ABSTRACT_INTERPRETER2)
 	javadoc --release $(JAVA_VERSION) --enable-preview -d $(dir $(DOC_JAVA)) $(JAVA_FILES_FOR_JAVA_DOC)
 
-$(BUILD_DIR)/generated/doc/unicode_version.adoc:
-	mkdir -p $(@D)
-	cd $(FZ_SRC) && git log modules/base/src/encodings/unicode/data.fz  | grep --extended-regexp "^Date:" | head | sed "s-Date:   -:UNICODE_VERSION: -g" | head -n1 > $$(realpath $(@D))/unicode_version.adoc
+$(BUILD_DIR)/generated/doc:
+	mkdir -p $@
+
+$(BUILD_DIR)/generated/doc/unicode_version.adoc: $(BUILD_DIR)/generated/doc
+	cd $(FZ_SRC) && git log modules/base/src/encodings/unicode/data.fz  | grep --extended-regexp "^Date:" | head | sed "s-Date:   -:UNICODE_VERSION: -g" | head -n1 > $(realpath $(@D))/unicode_version.adoc
 
 $(BUILD_DIR)/generated/doc/codepoints_white_space.adoc: $(CLASS_FILES_PARSER)
 	mkdir -p $(@D)

